apis/Google.Cloud.Logging.V2/Google.Cloud.Logging.Log4Net/LruCache.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
```

Comment:
Add license header ?  I am not certain if this is required. Seems to me every file has one, I assume that's the case.